app/deed/search/server.py

```diff
@@ -0,0 +1,6 @@
+from flask import redirect
+
+def register_routes(blueprint):
+    @blueprint.route('/deed/search', methods=['POST'])
```

Comment:
Any particular reason why this is in a separate file? Rather than putting it with it's respective GET function in the routes file?